runatlantis.io/docs/server-configuration.md

```diff
@@ -57,7 +57,7 @@ Values are chosen in this order:
 
   Notes:
   * Accepts a comma separated list, ex. `command1,command2`.
-  * `version`, `plan`, `apply`, `unlock`, `approve_policies`, `import` and `all` are available.
+  * `version`, `plan`, `apply`, `unlock`, `approve_policies`, `import`, `state` and `all` are available.
```

Comment:
Wouldn't this be state_rm?

Reply:
Or would state eventually unlock both state rm and state list commands?

Reply:
comment_parser now parse `atlantis state rm ...` as `command=state` and `subcommand=rm`. This allow-commands settings allow `command=state`, so it allows both state rm and state list and so on.

there are 3 ways to parse it.

1. `atlantis state rm ...` parsed with command=state and subcommand=rm
1. `atlantis state rm ...` parsed with command=state_rm
1. `atlantis state_rm ...` parsed with command=state_rm

I choose 1, cause it might be intuitive.

Reply:
got it, ty